actnow/petitions/models.py

```diff
@@ -28,6 +28,9 @@ class Petition(TimestampedModelMixin):
         _("problem statement"),
         max_length=1024,
     )
+    issue = models.OneToOneField(
+        "Issue", on_delete=models.DO_NOTHING, blank=True, null=True
+    )
```

Comment:
Not sure we should call this `issue` or `source` but:

1. Is it `OneToOne` thogh? It should be possible to create multiple petitions from the same issue/source e.g. one in support and another against.
2. Which capital `Issue` for verbose_name?

Reply:
@kilemensi  
I think in our conversations we had mentioned a petition should have 1 reason/issue/source but I think it's okay to allow multiple. I'll update this to `ManyToMany`

Reply:
Yes a petition should have 1 reason/issue/source but the same reason/issue/source can be used in multiple petitions.

Reply:
so A `ForeighnKey` relationship.